go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/go-resty/resty/v2 v2.7.0
 	github.com/gorilla/websocket v1.5.0
 	github.com/jarcoal/httpmock v1.3.0
+	github.com/polygon-io/ptime v1.3.0
```

Comment:
i don't think you need this, i don't see it being used anywhere, and its a private library